nginx.conf.sample

```diff
@@ -28,15 +28,15 @@ HA256:DHE-RSA-AES256-SHA256:DHE-DSS-AES256-SHA:DHE-RSA-AES256-SHA:!aNULL:!eNULL:
                 proxy_pass http://127.0.1.9:8000;
         }
 
-        location ~ ^/Dokumente/([^/]+)(/.*)? {
+        location ~ ^/Dokumente/([^/]+)/.*? {
                 auth_basic "Dokumente Admin Area";
-                auth_basic_user_file /var/run/Filer/Daten/Mandanten/$1;
+                auth_basic_user_file /var/run/Filer/Daten/Mandanten/users.pw;
```

Comment:
This is not the same thing.

I explicitly don't want all users to be able to access all web spaces, that's why nginx was supposed to make sure that the respective user, as matched by the regex, would provide the password for that very space … not ANY space. I wonder if that would be okay in your use cases, in mine it isn't.

Reply:
Ah, damn. You are right, obviously, that changes a lot.